docs/global/modules/core/pages/howto/mps-model-server-plugin.adoc

```diff
@@ -0,0 +1,41 @@
+= How-To run the light `model-server` in MPS as plugin
+:navtitle: mps-model-server-plugin
+
+
+You can run a *light version* of the `model-server` directly in MPS via the dedicated *MPS as Modelix Model Server Plugin*, which is published over at the https://plugins.jetbrains.com/plugin/22834-mps-as-modelix-model-server[JetBrains Marketplace^].
```

Comment:
Maybe we could create an explanation page that explains the difference between light and non-light model-server that can be linked here instead?

Reply:
sounds reasonable. but beyond this PR which provides a how-to only.

Reply:
Maybe we can at least lift the light-model-server information into a second how-to a la "how to use MPS as a model-server" or something like that?

Reply:
I am confused - this file is exactly what you suggest?